protozfits/digicam.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+from astropy.utils.decorators import lazyproperty
+
+from .simple import File
+from .patch_ids import PATCH_ID_INPUT, PATCH_ID_OUTPUT
+
+
+def event_source(path, run_id=0):
+    for event in File(path).Events:
+        yield Event(event, run_id)
+
+
+class Event:
+    def __init__(self, event, run_id):
+        self.run_id = run_id
+        self._event = event
+
+        _e = self._event
+        _w = self._event.hiGain.waveforms
```

Comment:
Because we reshape separetly `_.w_samples` and `_w.baselines` and sort `_w.pixelsIndices`. Might be better to  directly "reshape" `_w = _event.hiGain.waveforms` to speed it up. 
Is it possible?

Reply:
I am not sure I get the point here ...

Reply:
reshape very often returns a view into the same memory .. so reshape should be very fast.

Reply:
reshape seems to cost ~300ns